src/js/models/color.js

```diff
@@ -180,11 +181,11 @@ define(function (require, exports, module) {
     };
     
     /**
-     * Return string representation in CSS RGB format. 
-     * @return {string} Example: rgb(100, 100, 0)
+     * Return instance of TinyColor. Provide convenicen for accessing TinyColor functions
+     * @return {TinyColor} 
      */
-    Color.prototype.toCssRGB = function () {
-        return "rgb(" + [Math.round(this.r), Math.round(this.g), Math.round(this.b)].join(", ") + ")";
+    Color.prototype.toTinyColor = function () {
+        return tinycolor(this.toJS());
```

Comment:
Nvm, tinycolor sources say that they first make a percentage out of the color value and then multiply by 255, so they would be rounding in there. :+1: